stdlib/public/core/ContiguousArrayBuffer.swift

```diff
@@ -582,38 +570,18 @@ extension _ContiguousArrayBuffer {
 }
 
 /// This is a fast implementation of _copyToNativeArrayBuffer() for collections.
-///
```

Comment:
Benchmarking indicates that using `_UnsafePartiallyInitializedContigiousArrayBuffer` does not incur a measurable performance penalty. In fact, `addFrom` somehow compiles into faster code than the simple inline loop that's in the original code.